script/cibuild

```diff
@@ -35,6 +35,7 @@ output_fold() {
 function cleanup() {
   echo
   echo "::group::Shutting down services..."
+  docker compose logs db
```

Comment:
I added this because it helps a lot with debugging. It's a few extra log lines so figured we should just keep it instead of having to turn it on/off constantly.